### ISSUE TITLE
Add --stadepth to taup table documentation

### ISSUE THREAD

**Issue by atrabattoni** [crotwell/TauP#31]
It would be nice to be able to specify the station depth when computing travel time tables. An `--stadepth` argument could be passed to `taup_table` in the same way as `taup_time`.

**Comment by crotwell** [comment]
looking into the code, I believe this is actually already implemented, although undocumented.

```
taup table -ph P --stadepth 10
```

Please test and let me know if it doesn't work for you.

I will leave this ticket open to remember to add to the docs.

**Comment by atrabattoni** [comment]
Oh yeah you are right. I thought I'd tried this already... I edited the issue into a documentation problem. 

By the way: When downloading the 2.6.1 realease it looks like we get the 2.4.5 version. At least that's what `taup_table --version` is telling me (plus the fact that the new `taup table` syntax does'nt work.

**Comment by crotwell** [comment]
You may have more than one version of TauP installed. Can you check again and make sure you were running the version downloaded?

```
wget http://www.seis.sc.edu/downloads/TauP/TauP-2.6.1.tgz
tar zxf TauP-2.6.1.tgz 
./TauP-2.6.1/bin/taup table --version
edu.sc.seis:TauP:2.6.1 2022-07-25T15:53:32Z (748 204d077)
```

**Comment by atrabattoni** [comment]
You are right again. Someone else installed taup for everyone on the server I'm working on. My fault.

**Comment by crotwell** [comment]
fixed in version 3.0
